Library/Homebrew/cmd/update.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+
+if [ -z "$HOMEBREW_BREW_FILE" ]
+then
+  echo "Error: $(basename "$0") must be called from brew!" >&2
+  exit 1
+fi
+
+which_git() {
+  local which_git
+  which_git="$(which git 2>/dev/null)"
+  if [ -n "$which_git" ] && [ "/usr/bin/git" = "$which_git" ]
+    then
+    local active_developer_dir
+    active_developer_dir="$(/usr/bin/xcode-\select -print-path 2>/dev/null)"
+    if [ -n "$active_developer_dir" ] && [ -x "$active_developer_dir/usr/bin/git" ]
+    then
+      which_git="$active_developer_dir/usr/bin/git"
+    fi
+    fi
```

Comment:
One of these `fi` is indented too deeply.  And it looks as though the indenting of `then` is inconsistent in this function.